src/Publisher.ts

```diff
@@ -371,7 +385,7 @@ export default class Publisher {
         if (currentDepth >= 4) {
             return text;
         }
-
+		const published_files = await this.getFilesMarkedForPublishing();
```

Comment:
Can this be changed to camelCase to adhere to the convention in the project?
```suggestion
		const publishedFiles = await this.getFilesMarkedForPublishing();
```